### ISSUE TITLE
Plugins are not installing from Plugins Admin

### ISSUE THREAD

**Issue by soatrainer** [notepad-plus-plus/notepad-plus-plus#7981]
### Description of the Issue

In Old versions of the notepad++, I used to use Xml Tools plugin and jsonViewer Plugin. After upgrading to new versions from last couple of months, These plugins are not installing.

Operating System : Windows 10
Notepad++ version : 7.8.4

### Steps to Reproduce the Issue

1.  Launch Notepad++
2.  Go to Plugins --> Plugins Admin --> Select XML tools plugin --> click on install button 
3.   We will get a pop up saying , "The hash of Plugin package XML Tools is not correct, expected 'some value' found 'some other value' ".
 
### Expected Behavior
Plugin has to be installed successfully and ready to use.

### Actual Behavior
Plugin is not installing due to hash of the plugin.

### Debug Information

In Old versions ( a year back), All plugins used to work nicely, I installed a new version and facing problem from that time.

Regards,
Kishore

**Comment by soatrainer** [comment]
I have tried to installed the other plugins like dSpell it has same issue, but Auto save plugin was installed successfully.

**Comment by chcg** [comment]
@soatrainer Is this still happening with N++ 7.8.9? Update infos on plugins are updated together with N++. Data is managed at https://github.com/notepad-plus-plus/nppPluginList and checked with each modification. I'm not aware of problems like this for the last versions.

**Comment by siddharth-gitrepo** [comment]
Issue no longer present with v8.1.4.

**Comment by eroller** [comment]
> Issue no longer present with v8.1.4.

Still a problem for me. Trying to install JSONViewer.

**Comment by augustkst** [comment]
> Issue no longer present with v8.1.4.

Still a problem for me with Notepad++ 8.3.3. I'm trying to install DSpellCheck.

![image](https://user-images.githubusercontent.com/3762775/163866851-dab5e31f-5bfb-43e0-a2e1-16af8d9addfb.png)

**Comment by rpertusio** [comment]
In my case, this means there is a proxy (or some other intermediate system) that is modifying the response before it gets to Notepad++.

In my case:
* A corporate proxy was blocking the location of the plugin (a 3rd party site, not affiliated with NP++)
* The proxy was blocking with a "Are you sure you want to visit this site? [Continue]" page, which NPP 'downloaded' (instead of the '.zip' itself)
* The HTML 'continue?' page doesn't match a zip file, so NP++ stopped it.

Workaround:
* Go around the proxy, use a different network connection, or ask your IT Admin to unblock the specific plugin site

Feature Suggestion:
* NPP should permit the plugins to download and bypass the hash failure (at users' own risk)

**Comment by augustkst** [comment]
> In my case, this means there is a proxy (or some other intermediate system) that is modifying the response before it gets to Notepad++.

Indeed! After uninstalling Notepad++ and then installing it from the Company Portal application, the plugin can be installed.
